src/api/index.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export default {
+  getNewsItems() {
+    return axios.get('https://newsapi.org/v1/articles?source=the-next-web&sortBy=latest&apiKey=14d7c1ccdc5042c9b70b41aebde1348a');
+  },
```

Comment:
The trailing comma helps in achieving clarity in diffs. [Eslint rule](http://eslint.org/docs/rules/comma-dangle)